library/account/src/main/kotlin/org/cru/godtools/account/provider/okta/OktaModule.kt

```diff
@@ -61,19 +61,18 @@ internal abstract class OktaModule {
 
         @Provides
         @Singleton
-        fun OidcClient.oktaTokenStorage(
-            @ApplicationContext context: Context,
-            coroutineScope: CoroutineScope
-        ): TokenStorage = DataStoreTokenStorage(context).let { dataStore ->
-            when {
-                Build.VERSION.SDK_INT < Build.VERSION_CODES.M -> dataStore
-                else -> SharedPreferencesTokenStorage(this, context)
-                    .makeChangeAware()
-                    // migrate tokens if the user had previously used the DataStoreTokenStorage
-                    // TODO: this may race with loading tokens in the CredentialDataSource.
-                    //       We may want to create a "MergedTokenStorage" class instead to facilitate token migrations.
-                    .also { prefs -> coroutineScope.launch { dataStore.migrateTo(prefs) } }
+        fun OidcClient.oktaTokenStorage(@ApplicationContext context: Context): TokenStorage {
+            val dataStore = DataStoreTokenStorage(context)
+            if (Build.VERSION.SDK_INT < Build.VERSION_CODES.M) return dataStore
+
+            val sharedPreferences = try {
+                SharedPreferencesTokenStorage(this, context, verify = true)
+            } catch (e: Exception) {
+                Timber.tag(TAG).e(e, "Error creating Okta SharedPreferencesTokenStorage")
```

Comment:
We just log the exception for our records, we don't actually want the exception to crash the app